packages/model/system_states.go

```diff
@@ -1,29 +1,43 @@
 package model
 
-// SystemState is model
-type SystemState struct {
-	ID int64 `gorm:"primary_key;not null"`
+// Ecosystem is model
+type Ecosystem struct {
+	ID       int64 `gorm:"primary_key;not null"`
+	Name     string
+	IsValued bool
 }
 
 // TableName returns name of table
-func (ss *SystemState) TableName() string {
-	return "system_states"
+// only first ecosystem has this entity
+func (sys *Ecosystem) TableName() string {
+	return "1_ecosystems"
 }
 
 // GetAllSystemStatesIDs is retrieving all system states ids
 func GetAllSystemStatesIDs() ([]int64, error) {
-	states := new([]SystemState)
+	if !IsTable("1_ecosystems") {
+		return nil, nil
```

Comment:
why error is nil? This table is have to be created from first migration. If it isn't exists if should be logged.